plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineStandardFacet.java

```diff
@@ -206,7 +217,8 @@ public static IRuntime createAppEngineFacetRuntime(IProgressMonitor monitor)
   /**
    * Installs Java 1.7 facet if it doesn't already exist in <code>factedProject</code>
    */
-  private static void installJavaFacet(IFacetedProject facetedProject, IProgressMonitor monitor)
+  private static void addJavaFacetToBatchQueue(IFacetedProject facetedProject,
+      Set<IFacetedProject.Action> batchQueue)
```

Comment:
Is this a queue or a set? The name should match the type. Let's not call it a queue if it isn't. I was analyzing this code under the assumption it was FIFO, and now I realize that was wrong. 

same issue for the method name. Maybe just addJavaFacetToBatch?